fcrepo-indexing-solr/src/main/cfg/org.fcrepo.camel.indexing.solr.cfg

```diff
@@ -0,0 +1,31 @@
+# In the event of failure, the maximum number of times a redelivery will be attempted.
+error.maxRedeliveries=10
+
+# If the fedora repository requires authentication, the following values can be set:
+fcrepo.authUsername=
+fcrepo.authPassword=
+fcrepo.authHost=
+
+# The baseUrl for the fedora repository.
+fcrepo.baseUrl=localhost:8080/fcrepo/rest
+
+# The default `LDPath` transformation to use. This is overridden on a per-object
+# basis with the `indexing:hasIndexingTransformation` predicate.
+fcrepo.defaultTransform=default
+
+# If you would like to index only those objects with a type `indexing:Indexable`,
+# set this property to `true`
+indexing.predicate=false
+
+# The JMS connection URI, used for connecting to a local or remote ActiveMQ broker.
+jms.brokerUrl=tcp://localhost:61616
+
+# The camel URI for the incoming message stream.
+input.stream=activemq:topic:fedora
+
+# The baseUrl for the Solr server. If using Solr 4.x or better, the URL should include
+# the core name.
+solr.baseUrl=localhost:8983/solr/collection1
+
+# The timeframe (in milliseconds) within which new items should be committed to the solr index.
+solr.commitWithin=10000
```

Comment:
That was unintentional. I'll review the blueprint.xml files to see if anything else is missing